src/entities/trades/curve/CurveTrade.ts

```diff
@@ -482,11 +544,15 @@ export class CurveTrade extends Trade {
     invariant(chainId !== undefined && RoutablePlatform.CURVE.supportsChain(chainId), 'CHAIN_ID')
 
     try {
+      // Custom handler for xDAI<>USDT and xDAI<>USDC
+      // on Gnosis Chain
```

Comment:
remove before merging or elaborate on docstring

Reply:
Addressed in https://github.com/levelkdev/swapr-sdk/pull/80/commits/55595d108ce509867f961b739bfea556f9cc792f